docs/authenticate.md

```diff
@@ -62,18 +66,20 @@ You can use an access token to authenticate to the HERE platform and start worki
        customUrl: "http://YourCustomEnvironment",
        credentials: credentials,
        tokenRequester: requestToken,
+       expiresIn?: number;
      });
      ```
 
-   - For project authentication, specify the `tokenRequester` method, your project name, and credentials.
+   - For project authentication, specify the `tokenRequester` method, your project name, credentials, and the token expiration time.
```

Comment:
the same, it's optional

Reply:
Specified that it's optional in the note.